packages/preload/src/index.ts

```diff
@@ -119,7 +119,10 @@ export type LogType = 'log' | 'warn' | 'trace' | 'debug' | 'error';
 const originalConsole = console;
 const memoryLogs: { logType: LogType; date: Date; message: string }[] = [];
 
+export type FeedbackCategory = 'developers' | 'feature' | 'bug';
```

Comment:
Since we import this in the renderer, it should be in the packages/api under something like constants or something related to feedback.